src/Quizzer.API.DAL/Enums/AnswerFormat.cs

```diff
@@ -1,6 +1,7 @@
 ﻿namespace Quizzer.API.DAL.Enums {
-    public enum AnswerFormat : int { 
-        SingleChoice = 1, 
+    public enum AnswerFormat : int {
+        Undefined,
```

Comment:
What is the purpose of `Undefined`?

Reply:
Invalid fallback value which of course should not be used, I am using it in question facade during answering question evaluation. I will think about it if it is really needed, because I am using a switch case there. I will note this and try to refer the usage in question facade during new PR.